src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
https://github.com/rollbar/rollbar-php-laravel/blob/master/composer.json#L6

rollbar/rollbar-php-laravel is licensed like other Rollbar packages under MIT and was developed as part of the core Rollbar.com development effort, so you're free to use it like any other package